phpunit.xml.dist

```diff
@@ -19,7 +19,7 @@
   </testsuites>
 
   <filter>
-    <whitelist processUncoveredFilesFromWhitelist="true">
+    <whitelist processUncoveredFilesFromWhitelist="true" enablePathCoverage="true">
```

Comment:
This does not belong on the `<whitelist>` element. We do not, however, have an element for configuring generic code coverage settings like this. So the only place to put this for now would be the root `<phpunit>` element unfortunately.